src/views/anon/ventureBuildIntro/Home.tsx

```diff
@@ -27,6 +28,8 @@ import logo from '../../../assets/logo.svg'
 import laptop from '../../../assets/laptop.svg'
 import video from '../../../assets/video.svg'
 import mosaic from '../../../assets/mosaic.svg'
+import { orange } from '@mui/material/colors'
```

Comment:
might want to avoid using mui colours since they can change, maybe try a hardcoded hex value?

Reply:
yes, removed, using hardcoded hex value